src/selectors/per-thread/thread.js

```diff
@@ -116,8 +116,14 @@ export function getThreadSelectorsPerThread(threadIndex: ThreadIndex): * {
     }
   );
 
-  const getFilteredThread: Selector<Thread> = createSelector(
+  const _getTabFilteredThread: Selector<Thread> = createSelector(
```

Comment:
This only drops samples right? It doesn't modify stacks. I haven't really looked too closely at the implementation, but I think it's important to consider a couple of things:

* We can drop samples, and it doesn't really change any of our filter pipelines
* Change stack shapes is very dangerous, as it affects our CallNodePaths. If we plan to do this, we should think through where this happens in our data transform pipeline.
* If we put the tab filtering too late in the pipeline, it affects the thread activity graph.

For instance filtering by active tab produces:

<img width="857" alt="image" src="https://user-images.githubusercontent.com/1588648/70663928-fc964000-1c2e-11ea-8f38-d75e2791e781.png">

Which is great for the gecko engineer workflow, but not great for the webdev workflow. It would be better to show the graphs as if they were not doing work there.